### PR TITLE
Add API to list promotion memberships

### PR DESCRIPTION
**Pull request by Ray2126**

### CODE REVIEW

**Comment by seliwe** [comment]
@Ray2126 Can you deploy this to dev so we can preview changes?

**Comment by Ray2126** [comment]
Preview here: http://centrapay-docs.dev.s3-website-ap-southeast-1.amazonaws.com/

File: src/content/api/promotion-memberships.mdx [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+  path: API/Loyalty Programs
+  order: 4
+---
+import Properties from '../../components/Properties.astro';
+import Property from '../../components/Property.astro';
+import Endpoint from '../../components/Endpoint.astro';
+import Badge from '../../components/Badge.astro';
+import Error from '../../components/Error.astro';
+
+Promotion Memberships represents an Account's membership to a Promotion. Each Promotion Membership carries progress towards completing the Promotion. Promotion Memberships are created automatically for the account when a Loyalty Program Membership is created.
```
Comment (seliwe) [inline src/content/api/promotion-memberships.mdx:14]: Nice description :) you could include links to models for Account, Promotion, and Loyalty Program Membership

**Review by seliwe — APPROVED** [review]

File: src/content/api/promotion-memberships.mdx (hunk 1 above)
Comment (Ray2126) [reply src/content/api/promotion-memberships.mdx:14]: Yep, good idea :)

**Pull request merged**